todo_app/app.py

```diff
@@ -2,22 +2,29 @@
 import requests
 import os
 import datetime
+from bson.json_util import dumps
+from bson.json_util import loads
 from todo_app.flask_config import Config
 from todo_app.classModels import Item, ViewModel
+import pymongo
 
 def create_app(): 
     app = Flask(__name__)
     app.config.from_object(Config)
-
-    API_KEY = os.environ.get("api_key")
-    TOKEN = os.environ.get("token")
-    TRELLO_BOARD_ID = os.environ.get("TRELLO_BOARD_ID")
+    username=os.getenv("MONGO_USER")
+    password=os.getenv("MONGO_PASSWORD")
+    url=os.getenv("MONGO_URL")
+    database=os.getenv("MONGO_DB")
+    protocol=os.getenv("MONGO_PROTOCOL")
+    collection=os.getenv("MONGO_COLLECTION")
+    MONGO_URI="{0}{1}:{2}@{3}/{4}?retryWrites=true&w=majority".format(protocol,username,password,url,database)
```

Comment:
Note that the DB name specified in number 4 is only used if you access the default database, e.g. with:
```python
mongo = pymongo.MongoClient(MONGO_URI)
db = mongo.get_default_database()
collections = db.list_collection_names()
...
```

As it is your code currently creates/uses a database called "MyDatabase" for both your app & the tests, which is OK (you do use a different collection for your tests) but not quite what your code suggests